snorkel/slicing/sf.py

```diff
@@ -0,0 +1,22 @@
+"""Define SF appliers as simple alias of LF counterparts for now"""
+from typing import Callable
+
+from snorkel.labeling.lf import LabelingFunction, labeling_function
+
+
+class SlicingFunction(LabelingFunction):
```

Comment:
label_space is unused right now, would suggest a direct alias for both again. just for simplicity